[v6.1] Remaining edit pass

inertial-frame opened the pull request:
- [x] Finished up last items for: https://github.com/gravitational/teleport/issues/6534 in terms of setting more uniform conventions.
- [x] Corrected several HTML tables that weren't rendering.
- [x] Many typos corrected: mostly grammar, punctuation, formatting, etc.
- [x] Rewording in several sections.
- [x] Capitalized YML comments.
- [x] ~~More consistent use of Community Edition and Enterprise Edition throughout. Replaced the multiple ways we talked about the Community Edition.~~ Reverted and modified all references to the Community Edition.
- [x] Grammarly pass-through on all articles.

Good starting item that gave me a chance to walk through most of the docs set, learn, and go through several tutorials. At some point, we'd want to have a full editorial pass.

All of these changes will be ported to 6.2 and 7.0.

inertial-frame committed:
docs: improve enterprise rbac

inertial-frame committed:
docs: correct links and editions

inertial-frame committed:
docs: correct links and editions

inertial-frame committed:
docs: correct editions

inertial-frame committed:
docs: database access

inertial-frame committed:
docs: database access

inertial-frame committed:
docs: access controls

inertial-frame committed:
docs: more edits

inertial-frame committed:
docs: infra guides

inertial-frame committed:
docs: enterprise guides

inertial-frame committed:
docs: enterprise guides

inertial-frame committed:
docs: correct session tables

inertial-frame committed:
docs: cloud

inertial-frame committed:
docs: reference

inertial-frame committed:
docs: architecture

inertial-frame committed:
docs: linting fixes

inertial-frame committed:
docs: corrected

klizhentas commented on docs/pages/access-controls/getting-started.mdx:
```diff
@@ -64,10 +64,10 @@ she will be able to act as a system administrator and auditor at the same time.
 ## Step 2/3 Map SSO users to roles
 
-We are going to setup Github connector for OSS and Okta for Enterprises version.
+We're going to setup a Github connector for the Community Edition and Okta for Enterprise versions.
```
According to our website (which is our current source of truth for pricing and packaging), our editions are:

https://goteleport.com/pricing

* Open Source
* Pro 
* Enterprise

Our delivery modes are:

* Self-hosted (For Open Source and Enterprise editions)
* Cloud (For Pro and Enterprise editions)

klizhentas commented on docs/pages/adopters.mdx:
```diff
@@ -6,8 +6,9 @@ h1: Teleport Adopters and Use-Cases
 
-Whether you are using Teleport OSS, Enteprise or Cloud, tell us about your use-case.
-Just click "Improve the docs" below and send us a PR.
+Whether you're using Teleport Comunity, Enteprise or Cloud, tell us about your use-case.
```
Something like this will be more accurate: 

Whether you're using Teleport Open Source, Pro or Enterprise, self-hosted or in the cloud, tell us about your use-case

klizhentas commented on docs/pages/aws-oss-guide.mdx:
```diff
@@ -59,14 +59,14 @@ This guide will cover how to setup, configure and run Teleport on [AWS](https://
 ### EC2 / Autoscale
 
-To run Teleport in a HA configuration we recommend using m4.large instances. It's best practice to separate the proxy and authentication server, using autoscaling groups for both machines. We have pre-built AMIs for both Teleport OSS and Enterprise editions.
+To run Teleport in a HA configuration we recommend using m4.large instances. It's best practice to separate the proxy and authentication server, using autoscaling groups for both machines. We have pre-built AMIs for both Teleport Community and Enterprise editions.
```
Teleport Open Source

klizhentas commented on docs/pages/aws-oss-guide.mdx:
```diff
@@ -641,7 +639,7 @@ kube-system   kube-proxy-c5nv2           1/1     Running   0          5h28m
 ## Running Teleport Enterprise on AWS
 
-Most of this guide has been designed for OSS Teleport. Most of this guide also applies
+Most of this guide has been designed for the open source Teleport Community Edition. Most of this guide also applies
```
The Teleport Open Source edition

klizhentas commented on docs/pages/aws-terraform-guide.mdx:
```diff
@@ -150,7 +150,7 @@ cluster from scratch, so choose carefully.
 Setting `export TF_VAR_ami_name="gravitational-teleport-ami-ent-(=teleport.version=)"`
 
-Gravitational automatically builds and publishes OSS, Enterprise and Enterprise FIPS 140-2 AMIs when we
+Gravitational automatically builds and publishes Community, Enterprise, and Enterprise FIPS 140-2 AMIs when we
```
Teleport (our official company brand now) builds and publishes Open Source, Enterprsie, and ...

klizhentas commented on docs/pages/aws-terraform-guide.mdx:
```diff
@@ -191,7 +191,7 @@ The full local path to your Teleport license file, which customers can download
 Teleport Enterprise/Pro functionality.
 
-(OSS users can provide any valid local file path here - it isn't used by the auth server in a Teleport OSS install)
+(Community Edition users can provide any valid local file path here - it isn't used by the auth server in a Teleport Community Edition install)
```
Here and below Open Source Edition

klizhentas commented on docs/pages/faq.mdx:
```diff
@@ -92,27 +92,30 @@ Please refer to the [Ports](admin-guide.mdx#ports) section of the Admin Manual.
 ### Does Teleport support authentication via OAuth, SAML or Active Directory?
 
-Gravitational offers this feature for the [Enterprise versions of Teleport](enterprise/introduction.mdx).
+Gravitational offers this feature for all [Enterprise versions of Teleport](enterprise/introduction.mdx).
```
Teleport

klizhentas commented on docs/pages/kubernetes-access/getting-started.mdx:
```diff
@@ -317,7 +317,7 @@ We are going to setup Github connector for OSS and Okta for Enterprises version.
 
 <Tabs>
-  <TabItem label="Open Source">
+  <TabItem label="Community">
```
Here and below - "Open Source"

klizhentas reviewed (changes requested):
Almost there, just need to fix a couple of name references to match our website and marketing messaging.

inertial-frame committed:
Merge branch 'branch/v6.1' into adam/docs/61edits

inertial-frame committed:
docs: improved score for 6.1

inertial-frame committed:
docs: improve scores

inertial-frame committed:
docs: few more improvements

inertial-frame committed:
docs: fix merge conflict

inertial-frame commented:
@klizhentas - I reverted the use of Community Edition and updated existing references to the Open Source edition. Also applied a conversation score and grammar/spell checker to suss out the last items. Thanks!
